extension/content.js

```diff
@@ -12,8 +12,12 @@ const getUsername = () => $('meta[name="user-login"]').attr('content');
 function linkifyBranchRefs() {
 	$('.commit-ref').each((i, el) => {
 		const parts = $(el).find('.css-truncate-target');
-		let username = parts.eq(0).text();
-		let branch = parts.eq(1).text();
+		let branch = parts.eq(parts.length - 1).text();
+
+		// if there are two parts the first part is the username
+		if (parts.length > 1) {
+			let username = parts.eq(0).text();
+		}
```

Comment:
I'm confused, isn't that the same as on line 26-29?

See: https://github.com/sindresorhus/refined-github/commit/d17a13725e7823fb42916014cbd2944d36ee8eac